### PR TITLE
DFR-919 Update CaseEventToFields.json

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
Update FieldShowCondition for D8DivorceCostsClaim and D8FinancialOrder fields

### CODE REVIEW

**Review by rishikrsharma — APPROVED** [review]
why Banana was used?

**Comment by BWeeks-JusticeGovUK** [comment]
> 

I guess whoever wrote it really liked Bananas!?

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields.json [hunk 1]
```diff
@@ -1272,7 +1272,7 @@
     "PageID": "SolStatementOfTruth",
     "PageLabel": "Statement of truth and reconciliation",
     "PageDisplayOrder": 1,
-    "FieldShowCondition": "D8StatementOfTruth=\"Banana\"",
```
Comment (anatolysan) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields.json:1275]: I'm guessing the idea was not to show this label. ticket requests this to be shown?

Comment (anatolysan) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields.json:1275]: actually, I don't care

**Review by anatolysan — APPROVED** [review]

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields.json (hunk 1 above)
Comment (BWeeks-JusticeGovUK) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields.json:1275]: This one isn't required.

**Pull request merged**